test/runtime/CMakeLists.txt

```diff
@@ -5,7 +5,9 @@ function(halide_define_runtime_internal_test NAME)
 
     target_link_libraries(runtime_internal_${NAME} PRIVATE Halide::Runtime)
     set_target_properties(runtime_internal_${NAME} PROPERTIES ENABLE_EXPORTS TRUE)
-    target_compile_options(runtime_internal_${NAME} PUBLIC -Wno-builtin-declaration-mismatch )
+    if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+        target_compile_options(runtime_internal_${NAME} PUBLIC -Wno-builtin-declaration-mismatch )
+    endif()
```

Comment:
```suggestion
```
[This is not a valid Clang flag](https://gcc.godbolt.org/z/nnrvs6E19) and will print its own "warning: unknown warning option" warnings. We should really not be in the business of maintaining warning flag compatibility tables. Why are these warnings spurious, anyway?